plugins/serializers/binary/README.md

```diff
@@ -0,0 +1,133 @@
+# Binary Serializer Plugin
+
+The `binary` data format serializer serializes metrics into binary protocols using
+user-specified configurations.
+
+## Configuration
+
+```toml
+[[outputs.socket_writer]]
+  address = "tcp://127.0.0.1:54000"
+  metric_batch_size = 1
+
+  ## Data format to consume.
+  ## Each data format has its own unique set of configuration options, read
+  ## more about them here:
+  ## https://github.com/influxdata/telegraf/blob/master/docs/DATA_FORMATS_INPUT.md
+  data_format = "binary"
+  
+  ## Specify the endianness of the data.
+  ## Available values are "little" (little-endian), "big" (big-endian) and "host",
+  ## where "host" means the same endianness as the machine running Telegraf.
+  ## Default is "host".
+  endianness = "little"
+  
+  ## Definition of the message format and the serialized data.
+  ## Please note that you need to define all elements of the data in the
+  ## correct order.
+  ## An entry can have the following properties:
+  ##  read_from         --  Source of the data. Can be "field", "tag" or "time". 
+  ##                        If omitted "field" is assumed.
```

Comment:
What if a user wants to also encode the metric name?

Reply:
Yeah, I missed that, should be added with the latest commit